typings/vscode-mssql.d.ts

```diff
@@ -554,4 +555,12 @@ declare module 'vscode-mssql' {
         bindingType: BindingType;
         connectionStringSetting: string;
     }
+
+    export interface getAzureFunctionsParams {
+        filePath: string;
+    }
+
+    export interface getAzureFunctionsResult extends ResultStatus {
+        azureFunctions: string[];
```

Comment:
Something to consider for the future is making this return an object instead - that for right now can just contain a single name property. But it seems to me that it's pretty likely you'll want to get other data associated with a function eventually and this would make it a lot easier to work with later. 

But it's probably fine to just leave it like this for now anyways, since it also seems pretty unlikely that anyone else would be using this so updating later when that happens wouldn't be too hard.